tests/Feature/Generators/MigrationGeneratorTest.php

```diff
@@ -205,6 +205,7 @@ public function output_proper_pascal_case_model_names()
      */
     public function output_creates_constraints_for_unconventional_foreign_reference_migration()
     {
+        $this->app->config->set('blueprint.use_constraints', true);
```

Comment:
I don't understand why this is needed. Isn't the point of this PR to be able to say `use_constraints: true` in the YAML directly?